pmlc/dialect/stdx/tests/i1_to_i8.mlir

```diff
@@ -0,0 +1,31 @@
+// RUN: pmlc-opt -stdx-i1-storage-to-i8 %s | FileCheck %s
+
+module {
+  func @i1_to_i8_test(%arg0: memref<10x20xf32>, %arg1: memref<10x20xf32>) {
+    %cst = constant 0.000000e+00 : f32
+    %0 = alloc() : memref<10x20xi1>
+    // CHECK: %{{.*}} = alloc() : memref<10x20xi8>
+    %c0 = constant 0 : index
+    %c10 = constant 10 : index
+    %c1 = constant 1 : index
+    loop.for %arg2 = %c0 to %c10 step %c1 {
+      %c0_3 = constant 0 : index
+      %c20 = constant 20 : index
+      %c1_4 = constant 1 : index
+      loop.for %arg3 = %c0_3 to %c20 step %c1_4 {
```

Comment:
Use `%c0` and drop `%c0_3`. There seems to be redundant constants.

Reply:
Do we actually need loops for this test? I would shrink this test down to the bare essentials.